### DISCUSSION TITLE
MCD de la base

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#524] [General]
Serait il possible de disposer du MCD de la base. je souhairais analyser la cartographie avec un outil style PowerBI.
merci

**Comment by dbarzin** [comment]
Vous pouvez extraire le MCD en vous connectant directement à la base de données de Mercator.

**Comment by moiiom59** [comment]
Bonjour,
En effet je récupéré l’ensemble des schémas de tables mais je ne connais pas les relations ?

Bien cordialement,



***@***.***

Basset Laurent
RSSI du GHT
Cadre Adm. du pôle Médecine 2
***@***.***    03.28.28.54.73
***@***.***    06.43.34.31.37

  *   ***@***.******@***.***>
***@***.*** ***@***.***

Hôpital Alexandra Lepève
Centre Hospitalier de Dunkerque
Etablissement support du GHT du Dunkerquois et de l’Audomarois
130 av. Louis Herbeaux – CS 76367
59385 Dunkerque Cedex 1

***@***.***<https://bluefiles.com/chd/pagedepot-rssi>
Mon dépôt sécurisé

Vous souhaitez me transmettre des informations sensibles ou confidentielles ?
Cliquer sur le l’image ci-dessus pour accéder à mon dépôt sécurisé.








De : Didier Barzin ***@***.***>
Envoyé : jeudi 5 octobre 2023 18:00
À : dbarzin/mercator ***@***.***>
Cc : BASSET Laurent ***@***.***>; Author ***@***.***>
Objet : [*Newsletter*] Re: [dbarzin/mercator] MCD de la base (Discussion #524)


Vous pouvez extraire le MCD en vous connectant directement à la base de données de Mercator.

**Comment by dbarzin** [comment]
Tu peux utiliser la commande suivante pour extraire le schema :

    sudo mysqldump --no-data mercator  > mercator_schema.sql
